src/app/code/community/Allopass/Hipay/Helper/Data.php

```diff
@@ -616,6 +616,17 @@ public function getCheckoutSuccessPage($payment) {
 		return is_null(Mage::getStoreConfig('payment/'.$payment->getMethod().'/success_redirect_page')) ?
```

Comment:
The condition is_null() don't work, we have a problem in the sendRequest(). Prefer the function empty().